tests/Extensions/ExtensionManagerTest.php

```diff
@@ -141,15 +143,15 @@ public function test_boot_manager_with_one_manager_and_two_extensions()
         $this->container->shouldReceive('make')->with(ExtensionMock::class)->once()->andReturn(new ExtensionMock);
         $this->manager->register(ExtensionMock::class);
 
-        $this->container->shouldReceive('make')->with(ExtensionMock2::class)->once()->andReturn(new ExtensionMock2);
-        $this->manager->register(ExtensionMock2::class);
+        $this->container->shouldReceive('make')->with(\LaravelDoctrine\Tests\Mocks\ExtensionMock2::class)->once()->andReturn(new \LaravelDoctrine\Tests\Mocks\ExtensionMock2);
```

Comment:
It would be more consistent to import `\LaravelDoctrine\Tests\Mocks\ExtensionMock2` since `ExtensionMock` was imported.